lib/migration.js

```diff
@@ -1022,14 +1022,16 @@ function mixinMigration(MySQL, mysql) {
     return columnType;
   }
   function expectedColNameForModel(propName, modelToCheck) {
-    var mysql = modelToCheck.properties[propName].mysql;
-    if (typeof mysql === 'undefined') {
-      return propName;
-    }
-    var colName = mysql.columnName;
-    if (typeof colName === 'undefined') {
-      return propName;
+    if (modelToCheck.properties[propName].mysql) {
+      var mysql = modelToCheck.properties[propName].mysql;
+      if (typeof mysql === 'undefined') {
```

Comment:
I feel the logic here could be simplified:
`if(modelToCheck.properties[propName].mysql)` means `if (typeof mysql === 'undefined')` won't happen.

Or did I miss something?

Reply:
I am a bit confused what changes here:

Before: 
- If `modelToCheck.properties[propName].mysql` exists returns propName
- If `mysql.columnName` exists returns colName

And the new code also does the same thing...?

Reply:
You're right.. I just tried and couldn't to reproduce the issue 🙈 ... I think the original issue sets the relation incorrectly.